bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -661,13 +661,20 @@ private function addDocumentsNode(ArrayNodeDefinition $rootNode)
                 ->scalarNode('default_controller')
                     ->defaultValue('App\\Controller\\DefaultController::defaultAction')
                 ->end()
-                ->arrayNode('error_pages')
```

Comment:
This is acutally a BC break, we need to keep this as it is. 
I'd suggest to move the localized error pages under the `error_pages` node, so that it looks like: 
```yml
error_pages: 
    default: /foo/bar
    localized: 
        en: /en/error
        af: /af/error
```

Reply:
Adjusted as you requested.